src/query/execute_schema.c

```diff
@@ -11207,6 +11206,10 @@ build_attr_change_map (PARSER_CONTEXT * parser, DB_CTMPL * ctemplate, PT_NODE *
 	  break;
 	case PT_CONSTRAIN_NULL:
 	  attr_chg_properties->p[P_NOT_NULL] = ATT_CHG_PROPERTY_LOST;
+	  constr_att_list = cnstr->info.constraint.un.not_null.attr;
+	  chg_prop_idx = P_NOT_NULL;
+	  save_pt_costraint = true;
+	  break;
```

Comment:
How about handling it like below?

constr_att_list = NULL;
[[fallthrough]];

Reply:
I modified it as your opinion.